code/api/src/migrations/20200824211237-modify_products_add_new_fields.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return Promise.all([
+      queryInterface.addColumn('products', 'style', {
+        type: Sequelize.STRING
+      }),
+      queryInterface.addColumn('products', 'isSurvey', {
+        type: Sequelize.BOOLEAN,
+        defaultValue: false
+      })
+    ]);
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return Promise.all([
+      queryInterface.removeColumn('products', 'style'),
+      queryInterface.removeColumn('products', 'isSurvey')
+    ]);
+  }
+};
```

Comment:
Question for Alex: why did our migration work despite getting an error that a promise wasn't return and when we added return Promise.all it returned another error that it was already in the table?

Reply:
With migrations, we ran `npx sequelize-cli migration:create --name modify_users_add_new_fields` and this created a migration file and we went back to edit what we want.

Resource: https://dev.to/nedsoft/add-new-fields-to-existing-sequelize-migration-3527

Reply:
What's with the long numbers in the file name? @melatran @kiverso @Mariana-21

Reply:
@b-stein when we make migrations, it automatically makes a file with the time date stamp.
in rails, the migrations ran in order